Makefile

```diff
@@ -12,6 +12,12 @@ TAG = lorax-$(VERSION)-$(RELEASE)
 
 IMAGE_RELEASE = $(shell awk -F: '/FROM/ { print $$2}' Dockerfile.test)
 
+ifeq ($(TEST_OS),)
+TEST_OS = fedora-30
```

Comment:
Does the CI infrastructure defined TEST_OS for the various RHEL builds? 

Is it possible we have access to the latest RHEL snapshots/content repositories ?

Reply:
Yes. We're installing and subscribing those VMs in [rhel.setup]( https://github.com/cockpit-project/cockpit/blob/master/bots/images/scripts/rhel.setup). Cockpit only subscribes them during installation. For Composer, we probably want to resubscribe them to test pulling content from those repos. 

You can see all available images in Cockpit's [image-refresh](https://github.com/cockpit-project/cockpit/blob/master/bots/image-refresh#L28) script.